test/codegen/newman/fixtures/formdataFileCollection.json

```diff
@@ -1,44 +1,47 @@
 {
```

Comment:
you don't need to commit this file. It changes dynamically for adding the paths after creating the files for testing form data file upload

Reply:
Actully I committed the previous one half tested by mistake. So it didn't change automatically. I realised that later on.

Reply:
My point is you don't need to commit the file at all 😅

Reply:
Reverts to be the same as develop branch.